lib/msf/core/exploit/remote/kerberos/ticket/storage/base.rb

```diff
@@ -111,21 +135,21 @@ def objects(options, &block)
     #
     # @return [String] the info string
     def loot_info(options = {})
-      info = []
+      info = {}
 
       status = options.fetch(:status, :valid).downcase.to_sym
-      info << "status: #{status}" unless status == :valid
+      info[:status] = status unless status == :valid
 
       realm = options[:realm]
-      info << "realm: #{realm.to_s.upcase}" if realm.present?
+      info[:realm] = realm.to_s.upcase if realm.present?
 
       client = options[:client]
-      info << "client: #{client.to_s.downcase}" if client.present?
+      info[:client] == client.to_s.downcase if client.present?
 
       server = options[:server]
-      info << "server: #{server.to_s.downcase}" if server.present?
+      info[:server] = server.to_s.downcase if server.present?
 
-      info.join(', ')
+      JSON.generate(info) unless info.blank?
```

Comment:
```suggestion
      JSON.generate(info)
```

Reply:
If there's a reason why we need to return `nil` here, we should probably fix that